src/strategies/sumami-holders/index.ts

```diff
@@ -0,0 +1,69 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+import { strategy as erc20BalanceOfStrategy } from '../erc20-balance-of';
+
+export const author = 'arugulo';
+export const version = '0.1.0';
+
+// Merged ABI for sUMAMI and Marinate contracts
+const abi = [
+  'function balanceOf(address account) view returns (uint256)',
+  'function stakedBalance(address account, uint32 level) view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  if (options.marinateLevels.length > 4) {
+    return;
+  }
+  
+  const sUmamiBalances = await erc20BalanceOfStrategy(
```

Comment:
The strategy fails with the following error:
```
call revert exception (method="balanceOf(address)", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.4.0)

      at Logger.Object.<anonymous>.Logger.makeError (node_modules/@ethersproject/logger/src.ts/index.ts:213:28)

```
Looks like addresses in examples.json are not valid tokens/contracts on mainnet?

Reply:
Right it's on arbitrum. I've changed the network in the example